src/comm/bench-flow.js

```diff
@@ -331,14 +332,21 @@ module.exports.run = async function(configFile, networkFile) {
     //let configObject = require(absConfigFile);
     let configObject = Util.parseYaml(absConfigFile);
     let networkObject = require(absNetworkFile);
+    config.setConfigSetting('core:skipStartScript', config.getConfig()._config.argv().get('skipstart'));
+    config.setConfigSetting('core:skipEndScript', config.getConfig()._config.argv().get('skipend'));
```

Comment:
Line 335-336 is confusing, basically, you could set the same setting with two different names. `core:skip(Start/End)Script` will be automatically set when the `config` object is included/created. So please remove these lines.

I know `core:skip(Start/End)Script` is longer than writing `skipStart/End` in the command line, but Caliper will support many configuration points for many modules, so it's critical to identify them hierarchically. 

Fabric also has some really long settings names, but it's intuitive and maintainable :)